### PR TITLE
IS-14364 Support requires_service_api_access and use_webhook_secret in the manifest in Sesampy

### PR DESCRIPTION
**Pull request by parhambarazesh**
This PR adds the support for requires_service_api_access and use_webhook_secret in the manifest.
It also sets the permissions on the webhook pipes.

* Github Actions workflow is upgraded to use Ubuntu 20.04 (18.04 is unsupported from April 2023)

### CODE REVIEW

File: connector_cli/oauth2login.py [hunk 1]
```diff
@@ -59,6 +61,11 @@ def login_callback():
             "oauth_client_id": client_id,
             "oauth_client_secret": client_secret,
         }
+        if manifest.get("requires_service_api_access"):
+            secrets["service_api_access"] = service_jwt
```
Comment (branislavjenco) [inline connector_cli/oauth2login.py:65]: I think the secret in this case should be called `service_jwt`, like here https://github.com/datanav/connector-deployer/blob/0bb39b2aae70dec3e821ee657bfa7325b083e33c/service/service.py#L818

File: connector_cli/tripletexlogin.py [hunk 2]
```diff
@@ -32,6 +35,12 @@ def login_via_tripletex(sesam_node, args):
             secrets = {
                 "sessionToken": data["value"]["token"],
             }
+
+            if manifest.get("requires_service_api_access"):
+                secrets["service_api_access"] = args.service_jwt
```
Comment (branislavjenco) [inline connector_cli/tripletexlogin.py:40]: Same comment about the name of the secret.

File: connector_cli/oauth2login.py (hunk 1 above)
Comment (parhambarazesh) [reply connector_cli/oauth2login.py]: done

File: connector_cli/tripletexlogin.py (hunk 2 above)
Comment (parhambarazesh) [reply connector_cli/tripletexlogin.py]: done

**Review by branislavjenco — APPROVED** [review]

**Pull request merged**